src/flamegraph/color/mod.rs

```diff
@@ -339,6 +340,13 @@ pub(super) fn color(
         let reverse_name_hash = namehash(name.bytes().rev());
 
         (name_hash, reverse_name_hash, reverse_name_hash)
+    } else if deterministic {
+        use std::hash::Hasher;
+        let mut hasher = ahash::AHasher::default();
+        hasher.write(name.as_bytes());
+        let hash = hasher.finish() as f32 / u64::MAX as f32;
```

Comment:
This is probably going to give some weird rounding errors, since `f32`'s coverage over `u64` is quite bad. You may want to use `f64` for the casts, and then cast the resulting fraction to `f32`.